src/EPR.Accreditation.Facade/Services/AccreditationService.cs

```diff
@@ -255,5 +255,21 @@ private Common.Dtos.Accreditation CreateNewAccreditationDto(OperatorType operato
 
             return taskProgress;
         }
+
+        public async Task<HasOverseasAgentDto> GetHasOverseasAgent(Guid accreditationExternalId)
+        {
+            var hasOverseasAgent = await _httpAccreditationService.GetHasOverseasAgent(accreditationExternalId);
+
+            return hasOverseasAgent;
+        }
+
+        public async Task SetHasOverseasAgent(
+            Guid accreditationExternalId, 
+            bool? hasOverseasAgent)
+        {
+            var accreditation = await _httpAccreditationService.GetAccreditation(accreditationExternalId);
```

Comment:
We shouldn't have to perform a retrieval from the API to perform an update on the next line. What we should do is achieve a successful update from sending over an empty accreditation object that has the HasOverseasAgent set, when supplied with the relevant parameters